Assets/Plugins/Source/Core/Utility/FileUtility.cs

```diff
@@ -136,5 +203,29 @@ public static void CleanDirectory(string directoryPath, bool ignoreGit = true)
                 Debug.Log($"An error (which was ignored) occurred while cleaning \"{directoryPath}\": {ex.Message}");
             }
         }
+
+#if UNITY_EDITOR
+        public static void OpenFolder(string path)
```

Comment:
Small naming convention issue: I think this should be 'OpenDirectory' to match the rest of the usage.

Reply:
Fixed this.